devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy

```diff
@@ -1704,6 +1708,21 @@ def isYaleLock() {
 	return false
 }
 
+/**
+ * Utility function to check if the lock manufacturer is Samsung
+ *
+ * @return true if the lock manufacturer is Samsung, else false
+ */
+private isSamsungLock() {
+	if ("022E" == zwaveInfo.mfr) {
```

Comment:
Do you think we should add a null check for `zwaveInfo`?

Reply:
getZwaveInfo() returns `[:]` for the null case